steamapi.go

```diff
@@ -38,9 +41,10 @@ func (s SteamMethod) Request(data url.Values, v interface{}) error {
 	defer resp.Body.Close()
 
 	if resp.StatusCode != 200 {
-		return errors.New(fmt.Sprintf("Status code %d returned by %v", resp.StatusCode, string(s)))
+		return fmt.Errorf("steamapi %s Status code %d", s, resp.StatusCode)
 	}
 
 	d := json.NewDecoder(resp.Body)
-	return d.Decode(&v)
+	_ = d.Decode(&v)
```

Comment:
When there is an error at Steam level, it would return html, not json anymore, that's why I decided to ignore the decode.
What do you think?